cypress/support/config/services.js

```diff
@@ -421,7 +421,7 @@ const genServices = appEnv => ({
         smoke: false,
       },
       photoGalleryPage: {
-        path: isLive(appEnv) || isTest(appEnv) ? undefined : '',
```

Comment:
Having an empty string for the path makes the page load `http://localhost:7080`. It's best to leave it undefined.